internal/pkg/filesystem/fileloader/fileloader.go

```diff
@@ -177,18 +178,21 @@ func (l *loader) IsIgnored(path string) (bool, error) {
 	}
 	fileDef := filesystem.NewFileDef(filesystem.Join(path, KbcDirFileName))
 	fileDef.AddTag(`json`) // `json` is a constant model.FileTypeJson but cannot be imported due to cyclic imports, it will be refactored
-	if l.fs.Exists(fileDef.Path()) {
```

Comment:
Problem bol tu, `path` je `kbcdir.json`, ale na disku je `kbcdir.jsonnet`.
Ale o tom vie az `fileloader` a `mapper`, takze sa neexistencia suboru testuje nizsie `if errors.Is(err, filesystem.ErrNotExist)`.

To ukazuje, ze tento system musim lepsie premysliet, kedze sa da pouzit nespravnym sposobom.
Testy lokalne presli, tak snad je to vsetko.

Reply:
ómg, aha! Díky. 👍